CMakeLists.txt

```diff
@@ -92,7 +92,8 @@ option( BUILD_SHARED_LIBS "Build hipBLAS as a shared library" ON )
 # Find CUDA if the user wants a CUDA version.
 option(USE_CUDA "Look for CUDA and use that as a backend if found" OFF)
 if (USE_CUDA)
-    find_package( CUDA REQUIRED )
+	set (ENV{CUDA_BIN_PATH} "${CUDA_PATH}" )
+	find_package( CUDA REQUIRED )
```

Comment:
I would suggest leaving this bit unchanged and setting `CUDA_BIN_PATH` via `install.sh`. That way, experienced users of CMake who are familiar with the standard ways of customizing the search paths for CUDA will be able to build using CMake directly using options they're already familiar with.

You actually do already set `CUDA_BIN_PATH` in the install script, but I think you may have forgotten to export it.